.github/workflows/pre-release-continuous-test.yaml

```diff
@@ -9,15 +9,24 @@ on:
   push:
     branches:
       - pre-release-continuous-test
+      # remove this before landing PR
+      - rustielin/forge-multi-nightly
   schedule:
-    # Run hourly
-    - cron: "0 * * * *"
+    # Run every 3 hours
+    - cron: "0 */3 * * *"
```

Comment:
I would say let's make this even longer - may be 6 hours? This will give us enough time for run a longer running performance test for may be 4-5 hours.

Reply:
The test suites today don't take that long. It's running performance benchmark rn which only runs for 5 min. We can tune this later ofc, but I'll land this for now